cds/modules/previewer/templates/cds_previewer/macros/player.html

```diff
@@ -72,11 +83,29 @@
            }
        ];
       {% endif %}
+      {%- if embed_config.autoplay -%}
+      player.autoplay = true;
+      {% endif %}
+      // not working currently
+      {%- if embed_config.loop -%}
+      player.loop = true;
```

Comment:
does this break anything since it is not supported or it is ok?

Reply:
It does not break anything, that's why I left it for future